pandas/core/generic.py

```diff
@@ -3325,30 +3330,35 @@ def to_latex(
             tabular output.
         DataFrame.to_html : Render a DataFrame as an HTML table.
 
+        Notes
+        -----
+
+        .. note::
```

Comment:
IMO this seems redundant with the note heading section

Reply:
fair point will look to remove